core/producer_server_process.py

```diff
@@ -131,7 +130,8 @@ def start(self, debug: bool) -> None:
                 sys.exit()
 
     def process_action(self, action: str, client: socket.socket) -> None:
-        """
+        """Perform actions depending on the flow of file.
```

Comment:
Codacy found an issue: [Multi-line docstring summary should start at the second line (D213)](https://app.codacy.com/manual/excalibur.krv/m3u8-dl/pullRequest?prid=6274429)